src/mbedtls.c

```diff
@@ -94,7 +116,7 @@ _libssh2_mbedtls_safe_free(void *buf, int len)
 
 #ifdef LIBSSH2_CLEAR_MEMORY
     if(len > 0)
-        memset(buf, 0, len);
+        memset_func(buf, 0, len);
```

Comment:
The whole static-volatile-memset-fptr should already handled by `_libssh2_explicit_zero`.